code/modules/paperwork/ticketmachine.dm

```diff
@@ -13,11 +13,13 @@
 	maptext_y = 10
 	layer = HIGH_OBJ_LAYER
 	var/ticket_number = 0 //Increment the ticket number whenever the HOP presses his button
-	var/current_number = 0 //What customer are we currently serving?
-	var/max_number = 999 //To stop the text going fucky. At this point, you need to refill it.
-	var/cooldown = 50 //Small cooldown, stops the clown from immediately breaking it.
+	var/current_number = 0 //What ticket number are we currently serving?
+	var/max_number = 100 //At this point, you need to refill it.
+	var/cooldown = 50 //Small cooldown, used when emagged to prevent flaming ticket spam
 	var/ready = TRUE
 	var/id = "ticket_machine_default" //For buttons
+	var/list/ticket_holders = list()
+	var/list/tickets = list()
```

Comment:
Travis is failing because you're calling a proc on an item in a list without a type specifically defined, this doesn't technically matter, but Travis isn't happy with it.
I can see two easy ways to fix it, either define a type for the list (`var/list/obj/tickets = list()` or whatever is required, or set another variable to hold the current ticket;
```
var/obj/current_ticket = tickets[current_number]
current_ticket.visible_message()
```

Reply:
Thanks!